supervisor-services-labs/argocd-operator/argocd-operator.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: argocd-operator.fling.vsphere.vmware.com
+spec:
+  categories:
+  - CI/CD
+  - GitOps
+  displayName: argocd-operator
+  iconSVGBase64: none
+  longDescription: The Argo CD Operator manages the full lifecycle for Argo CD and
+    its components. The operator's goal is to automate the tasks required when operating
+    an Argo CD cluster. Beyond installation, the operator helps to automate the process
+    of upgrading, backing up and restoring as needed and remove the human as much
+    as possible. In addition, the operator aims to provide deep insights into the
+    Argo CD environment by configuring Prometheus and Grafana to aggregate, visualize
+    and expose the metrics already exported by Argo CD.
+  maintainers:
+  - name: vSphere IaaS and Consumption
```

Comment:
Note: we should actually get a DL so that customers can send us feedback? WDYT?

Reply:
Yup. Let me get a DL for this. Will make you an admin